src/main/java/fish/payara/extras/upgrade/BaseUpgradeCommand.java

```diff
@@ -125,9 +121,11 @@ protected void validate() throws CommandException {
 
         // Set up the install dir variable
         glassfishDir = getInstallRootPath();
+        logger.log(Level.FINEST, "glassfishDir resolved as {0}", glassfishDir);
```

Comment:
Using inherited logger will report wrong file in the log.
I see the usage even in LocalDomainCommand, it is wrong as well.

Reply:
The log output is correct, it just doesn't follow logger setup for this class or package, it follows rules for com.sun.enterprise.admin.servermgmt.cli.
But if it is done this way for all commands, let's discuss it elsewhere.